core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -1328,6 +1328,15 @@ private void processResponse(org.jgroups.Address src, byte[] buffer, int offset,
          requests.addResponse(requestId, address, response);
       } catch (Throwable t) {
          log.errorProcessingResponse(requestId, src);
+         Exception e = t instanceof Exception ? ((Exception) t) : new CacheException(t);
+         try {
+            marshaller.objectFromByteBuffer(buffer, offset, length);
+         } catch (IOException e1) {
+            e1.printStackTrace();  // TODO: Customise this generated block
```

Comment:
Don't forget the TODOs here !

Reply:
Ouch, thanks - this was a debugging code ofc :-/